ui/src/app/components/inputs.tsx

```diff
@@ -250,7 +249,6 @@ export const TextAreaWithLengthValidationMessage = (
         value={text}
         onBlur={() => updateShowTooShortWarning()}
         onChange={(v) => onTextUpdate(v)}
-        ariaLabel={ariaLabel}
```

Comment:
I introduced ariaLabel in https://github.com/all-of-us/workbench/pull/8511/files#diff-e20d25e18ad70d1c526af33dd200e80376f4c624abc466a553a1563a9c0eb3d7R165-R253